src/plays/git-hub-profile-search/styles.css

```diff
@@ -0,0 +1,177 @@
+/* enter stlyes here */
+.play-heading {
```

Comment:
Prefix all the styles with a unique style name.

Reply:
ok